### ISSUE TITLE
Expose hands and controllers at the same time

### ISSUE THREAD

**Issue by cabanier** [immersive-web/webxr-hand-input#120]
The Quest platform is able to provide information on hands and controllers at the same time.
Should we expose this to WebXR? If so, should we create a new feature string?

/tpac expose hands and controllers at the same time

**Comment by Manishearth** [comment]
This would mean that the XRInputSource just has both a gamepad and hand object on it, yes? We can add language to that effect.

I don't think this needs a new feature, it's a part of hand input.

**Comment by cabanier** [comment]
the positions of the hand and the controller would be different. It's not just that you get the controller inputs.

**Comment by Manishearth** [comment]
Ah, hmm. Interesting.

**Comment by cabanier** [comment]
yes, this is why it only works with Quest Pro controllers. They are self tracked so they don't need IR emitters and HMD camera tracking.

**Comment by De-Panther** [comment]
Apparently the ML2 browser already exposes hands and the controller at the same time.
As the `XRSession.inputSources` is a list of active input sources, it can contain more than 2 sources.
